src/js/main.js

```diff
@@ -14,8 +14,8 @@ Site = {
 
     $(document).ready(function () {
       Site.Shapes.init();
-      Site.Map.init();
       Site.Minimap.init();
+      Site.Map.init();
```

Comment:
Why did you revert this? Minimap depends on Map and should be init after anyway?